### PR TITLE
[full-ci] Adapt to new login design

### PR DESCRIPTION
**Pull request by hurradieweltgehtunter**
fixes: https://github.com/owncloud/twofactor_totp/issues/237
fixes: https://github.com/owncloud/twofactor_totp/issues/231

### CODE REVIEW

**Review by DeepDiver1975 — COMMENTED** [review]
:robot: :red_circle:

**Comment by hurradieweltgehtunter** [comment]
I don't get it. Why is the drone build always killed?

**Comment by JammingBen** [comment]
@hurradieweltgehtunter This acceptance test fails: https://drone.owncloud.com/owncloud/twofactor_totp/1372/14/11

```
Fatal error: Call to a member function click() on null (Behat\Testwork\Call\Exception\FatalThrowableError)
```

The xpath for the submit action is wrong now: `'//form/input[@type="submit"]'` (see https://github.com/owncloud/twofactor_totp/blob/master/tests/acceptance/features/lib/VerificationPage.php#L35).

The other tests were probably cancelled due to this.

**Comment by JanAckermann** [comment]
Will take care about the tests soon

**Comment by phil-davis** [comment]
I put `full-ci` in the PR title so that all test pipelines will run to completion. That will help to see all the tests that fail, in one pass.

**Comment by JanAckermann** [comment]
@DeepDiver1975 @hurradieweltgehtunter  CI should be green now, please review and merge, thx! 
We might need to create a release issue as well

**Comment by prsnbrg** [comment]
What is the current status?

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Tested with the following apps:
 
 ## Enabling TOTP 2FA for your account
 ![](https://raw.githubusercontent.com/owncloud/twofactor_totp/stable9.1/screenshots/settings.png)
-![](https://raw.githubusercontent.com/owncloud/twofactor_totp/stable9.1/screenshots/verify.png)
+![](https://raw.githubusercontent.com/owncloud/twofactor_totp/stable9.2/screenshots/verify.jpg)
```
Comment (phil-davis) [inline README.md:14]: The reference to `stable9.2` does not look correct. How will this work after merge of this PR?

Comment (mmattel) [reply README.md:14]: Just checked if the link (old and new) returns an image but only does a 404

**Review by mmattel — APPROVED** [review]

**Pull request merged**